awx/ui/src/components/JobList/useWsJobs.js

```diff
@@ -31,7 +31,7 @@ export default function useWsJobs(initialJobs, fetchJobsById, qsConfig) {
         return;
       }
       setJobsToFetch([]);
-      const newJobs = await fetchJobsById(throttledJobsToFetch);
+      const newJobs = await fetchJobsById(throttledJobsToFetch, {});
```

Comment:
@kialam just a minor suggestion we can rely on `qs` default value of `fetchJobsById`.
```suggestion
      const newJobs = await fetchJobsById(throttledJobsToFetch);
```